tools/load_veldwerk_json.py

```diff
@@ -423,8 +425,52 @@ def fielddata_to_memcollections(filename, profile_plan_col=None, profile_id_fiel
 
             log.warning('records toegevoegd %i', i + 1)
 
+    # Extract fixed points from the geojson
+    for fp_pk, point_ids in enumerate(json_dict['point_notes']):
+
+        fp = json_dict['point_notes'][point_ids]
+        fixed_point = OrderedDict()
+
+        fixed_point['vp_pk'] = fp_pk
+        fixed_point['ids'] = fp.get('id', '')
+        fixed_point['project_id'] = project_id
+        fixed_point['proj_name'] = proj_name
+        fixed_point['type'] = fp.get('note_type', '')
+        fixed_point['opm'] = fp.get('remarks', '').replace('\n', '')
+
+        photo_list = fp.get('photos', None)
+        photo_string = ''
+
+        if photo_list:
+            for i in range(len(photo_list)):
+                f_name = os.path.basename(photo_list[i]).split('.')[0]
+                if i != (len(photo_list) - 1):
+                    photo_string = photo_string + (f_name + ";")
+                else:
+                    photo_string = photo_string + f_name
+
+        fixed_point['fotos'] = photo_string
+
+        fixed_point['datum'] = fp.get('datetime', '')
+
+        if fp.get('rd_coordinates') and not None in fp.get('rd_coordinates')[:2]:
```

Comment:
In de nieuwste versie zit alle informatie over locatie in 'locations' en is er een x y en altitude_nap_pole attribute (z is de hoogte van de gps sensor, hier wordt nog de stok lengte en L1 lengte (afstand tussen stok en gps meter in het apparaat) afgehaald).
ik zou daarom een regel hierboven schrijven waar de x, y, altitude_nap_pole bepaald op basis van de 'location' als die aanwezig is of op basis van de rd_coordinaten[0], rd_coordinaten[1] en altitude_nap_pole.

Reply:
Dit geldt alleen voor de point_notes of ook voor de rest van de json?

Reply:
dit geldt ook voor de profielen

Reply:
Is hier al een voorbeeld van? Ik vind het een beetje lastig om het zo voor me te zien in het geval van de profielen

Reply:
Heb de code aangepast aan de nieuwe versie